test/src/actions/ios/welcome.actions.js

```diff
@@ -13,7 +13,8 @@ class WelcomeActions extends AndroidWelcomeActions {
         let wordText2 = '';
         let wordText3 = '';
         for (let i = 1; i <= 8; i++) {
-            wordList = await $$(Pages.welcomePage.wordList);
+            await ActionHelper.waitForDisplayed(Pages.welcomePage.nemonic, 3000);
+            wordList = await $$(Pages.welcomePage.nemonic);
```

Comment:
we can encapsulate these lines in wdio-common in a method to return a list of elements

on the other hand, small typo "mnemonic" instead of "nemonic" ;)